shared/src/state-migrations/migration-functions.ts

```diff
@@ -36,7 +36,7 @@ export interface Migration {
 }
 
 export const migrations: {
-    [key: number]: Migration;
+    [CurrentStateVersionToMigrateTo: number]: Migration;
```

Comment:
I think this would rather be `StateVersionToMigrateTo` as the current version before applying a migration is one less than the specified number. But also not that important.

Reply:
The property in the `State` is called `currentStateVersion` (See `shared\src\state.ts`). Therefore this name.
I guess we should have just named it `stateVersion` or even `version` instead.

What do you think about:

```suggestion
    [TargetStateVersion: number]: Migration;
```